packages/periphery/test/CurveOracle.spec.ts

```diff
@@ -130,28 +136,26 @@ describe('CurveEMAPriceOracle', () => {
     );
 
     await expect(
-      oracle.addPool(constants.AddressZero, baseToken.address, quoteToken.address)
+      oracle.addPool(constants.AddressZero, baseToken.address, quoteToken.address, false)
     ).to.be.revertedWithCustomError(oracle, 'ZeroAddress');
-    await expect(oracle.addPool(pool.address, constants.AddressZero, quoteToken.address)).to.be.revertedWithCustomError(
-      oracle,
-      'ZeroAddress'
-    );
-    await expect(oracle.addPool(pool.address, baseToken.address, constants.AddressZero)).to.be.revertedWithCustomError(
-      oracle,
-      'ZeroAddress'
-    );
-    await expect(oracle.addPool(pool.address, baseToken.address, baseToken.address)).to.be.revertedWithCustomError(
-      oracle,
-      'InvalidTokenAddress'
-    );
-    await expect(oracle.addPool(pool.address, baseToken.address, anotherToken.address)).to.be.revertedWithCustomError(
-      oracle,
-      'InvalidTokenAddress'
-    );
-    await expect(oracle.addPool(pool.address, anotherToken.address, quoteToken.address)).to.be.revertedWithCustomError(
-      oracle,
-      'InvalidTokenAddress'
-    );
+    await expect(
+      oracle.addPool(pool.address, constants.AddressZero, quoteToken.address, false)
+    ).to.be.revertedWithCustomError(oracle, 'ZeroAddress');
+    await expect(
+      oracle.addPool(pool.address, baseToken.address, constants.AddressZero, false)
+    ).to.be.revertedWithCustomError(oracle, 'ZeroAddress');
+    await expect(
+      oracle.addPool(pool.address, baseToken.address, baseToken.address, false)
+    ).to.be.revertedWithCustomError(oracle, 'InvalidTokenAddress');
+    await expect(
+      oracle.addPool(pool.address, baseToken.address, anotherToken.address, false)
+    ).to.be.revertedWithCustomError(oracle, 'InvalidTokenAddress');
+    await expect(
+      oracle.addPool(pool.address, anotherToken.address, quoteToken.address, false)
+    ).to.be.revertedWithCustomError(oracle, 'InvalidTokenAddress');
+    await expect(
+      oracle.addPool(pool.address, quoteToken.address, baseToken.address, true)
```

Comment:
Is this test about that `_getPriceX96` check? Doesn't this tx gets reverted with something like `function selector is not recognized: *hash*`? Maybe we should make the corresponding revert reason check: those hashes are independent of contract instances and methods implementation (unless you change either method name or argument types/order), so you even can hardcode them

Reply:
And seems like we are missing the symmetrical test (with a `false` arg)